smart-hadoop-support/smart-hadoop/src/test/java/org/smartdata/hdfs/action/TestCopyFileAction.java

```diff
@@ -122,38 +142,11 @@ public void testRmoteCopyWithOffset() throws Exception {
     }
   }
 
-  /*@Test
-  public void testLocalCopyWithOffset() throws Exception {
```

Comment:
Can you clarify the purpose of this change?

Reply:
The local and remote versions of these tests differ only in the destination path prefix (absolute HDFS path in the remote case or relative path in the local case), so I simply introduced parameter and deleted duplicated code. However, I decided to completely refactor this test and now separate target HDFS mini-cluster is used in remote case (as it was done to test `distcp` action)